docs/browser-lifecycle.md

```diff
@@ -0,0 +1,74 @@
+# Browser Lifecycle Events
+
+This document defines a set of browser events organized into three high-level lifecycle phases.
+
+It focuses on the **purpose and relationships of events within the browser lifecycle**, rather than defining their individual attributes (which are specified in their respective semantic convention documents).
+
+---
+
+## Goals
+
+The purpose of this document is to:
+
+- Provide a **conceptual lifecycle model** for browser telemetry.
+- Define **which events are expected in each phase** of a page lifecycle.
+- Help instrumentation authors and data consumers **reason about browser behavior from load to unload**.
+- Clarify the **purpose and relationships of events without defining their individual attributes**, which are specified in their respective semantic convention documents.
+
+This structure ensures that browser-related events can be consistently interpreted across implementations.
+
+---
+
+## Lifecycle
+
+The browser lifecycle is divided into the following phases:
+
+1. **Load phase** – from navigation start to Largest Contentful Paint (LCP)
+2. **User interaction phase** – from LCP until the page begins unloading
+3. **Unload phase** – when the page begins unloading
+
+---
+
+## Lifecycle Phases
+
+### Load Phase
+
+**Starts at navigation start → Ends at Largest Contentful Paint (LCP)**  
+Captures the loading behavior and performance of the page.
+
+| Event | Description | Semantic Conventions Status | Instrumentation Status                                                                                                                               |
+|---|---|---|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `browser.page_view` | Signals the start of a **hard** page navigation. Occurs once per page load. | In review [PR1910](https://github.com/open-telemetry/semantic-conventions/pull/1910) | In review [PR2386](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2386)                                                             |
+| `browser.navigation_timing` | Captures detailed technical milestones from the [PerformanceNavigationTiming](https://developer.mozilla.org/docs/Web/API/PerformanceNavigationTiming) API. Occurs once per page load. | In review [PR1919](https://github.com/open-telemetry/semantic-conventions/pull/1919) | Not created                                                                                                                                          |
+| `browser.page_load` | Summarizes the user-visible page load experience from navigation start until LCP (includes total duration, paint timings, resource counts, transfer size). Occurs once per page load. | Not created | Not created                                                                                                                                          |
```

Comment:
Is the main intent of this event to summarize all the other events that happen during page load to make it easier for backends?

Alternatively
- duration is part of navigation timing (if based on the window load event)
- FCP and LCP are part of web vitals
- resource counts could be determined from resource timing data
- transfer size is available in resource timing data